nilearn/datasets/description/nki_enhanced_surface.rst

```diff
@@ -14,21 +15,24 @@ Enhanced Nathan Kline Institute-Rockland Sample resting state fMRI data
 according to https://github.com/fliem/nki_nilearn.
 Contains phenotypical metadata.
 
+Direct download link :footcite:t:`NKIdataset`.
+
```

Comment:
Maybe it's better to directly put the link here
```suggestion
Direct download link: http://fcon_1000.projects.nitrc.org/indi/enhanced/
```
It rendered like this and it wasn't super obvious to me that I have to click the little 1
<img width="451" alt="Screenshot 2025-01-15 at 1 50 56 PM" src="https://github.com/user-attachments/assets/2d742129-b481-4ae6-adb2-fa6c4cbfda18" />